README.md

```diff
@@ -13,16 +13,53 @@ Inspired by the [invidio.us](https://github.com/omarroth/invidious) project.
 
 ## Installation
 
+You need to install nimble on your system: https://nim-lang.org/install.html
```

Comment:
Nim, not nimble. nimble is Nim's package manager which is used to build the project to pull in dependencies.